project/README.md

```diff
@@ -15,7 +15,20 @@
 
 ## React
 
-## Swift
+## Swift   
+[Tanay Khandke](https://github.com/tkhandke)
+
+**Major**
+
+* Documentation -   
+tips for swift unit testing on linux - https://github.com/tkhandke/swift_doc
+
+**Minor**
+
+* research
```

Comment:
For each of these three, either include links that show evidence of your work or cut them out.